backend/hitas/views/housing_company.py

```diff
@@ -153,6 +153,8 @@ def update(self, instance: HousingCompany, validated_data: dict[str, Any]):
         return instance
 
     def validate_business_id(self, value):
+        if value == "":
+            return None
```

Comment:
is there a reason to convert `""` to `None`? If not, updating `validate_business_id` seems more reasonable

Reply:
This way we allow the frontend to pass `""` as a valid value instead of `null`, and keep our data cleaner (Either a valid business id value or `null`)

Other option would be to allow all `""` `null` and a valid business id, and change the field to accept blank values (currently it doesn't accept them)

Reply:
ok, so there's a reason to convert `""` to `None`. For me the confusing part here is the different handling of `None` and `""`:

 - pass `""` -> return `None`
 - pass `None` -> call `validate_business_id(None)` -> return `None`
 - anything else `validate_business_id(value)` -> `raise` or `return <value>`
 
Maybe this would be bit more clear:

```
if value == "":
    value = None
validate_business_id(value)
return value
```

This way it's more clear that `""` is converted to `None` on purpose and same logic applies to all values.